themes/digital.gov/layouts/communities/list.html

```diff
@@ -23,33 +23,108 @@ <h1>{{- .Title | markdownify -}} </h1>
   </div>
 
 
-  <section class="communities-list usa-section">
+  <section class="usa-section">
     <div class="grid-container-desktop">
       <div class="grid-row tablet-lg:grid-gap-2">
+          <div class="grid-col-12">
+            <ul class="usa-card-group">
+              {{/* Gets all communities pages */}}
+              {{ $dg_communities := (where .Site.Pages "Section" "communities").ByTitle }}
+
+              {{/* Gather only the pages with a weight greater than or equal to 1 */}}
+              {{ $dg_communities := (where $dg_communities ".Params.weight" "ge" 1) }}
+
+              {{/* Gather only the pages with a dg-emphasis key */}}
+              {{ $dg_communities := (where $dg_communities ".Params.dg-highlight" "eq" true) }}
+
+              {{ range $i, $e := $dg_communities }}
+              <!-- <li class="usa-card tablet-lg:grid-col-4">
+                <div class="usa-card__container">
+                  <div class="usa-card__header">
+                    <h2 class="usa-card__heading">{{- $e.Title | markdownify -}}</h2>
+                  </div>
+                  <div class="usa-card__body">
+                    <p>{{- $e.Params.summary | markdownify -}}</p>
+                  </div>
+                  <div class="usa-card__footer">
+                    <a href="{{- $e.Permalink -}}" title="{{- $e.Title | markdownify -}}">
+                      <button class="usa-button">Join Now!</button>
+                    </a>
+                  </div>
+                </div>
+              </li> -->
+
+              <li class="usa-card usa-card--flag tablet-lg:grid-col-12">
+                <div class="usa-card__container">
+                  <div class="usa-card__header">
+                    <h2 class="usa-card__heading">{{- $e.Title | markdownify -}}</h2>
+                  </div>
+                  <div class="usa-card__media">
+                    <div class="usa-card__img">
+                      <img src="{{- $e.Params.dg_image | absURL -}}" alt="{{- $e.Title }}">
+                    </div>
+                  </div>
+                  <div class="usa-card__body">
+                    <p>{{- $e.Params.summary | markdownify -}}</p>
+                  </div>
+                  <div class="usa-card__footer">
+                    <a href="{{- $e.Permalink -}}" title="{{- $e.Title | markdownify -}}">
+                      <button class="usa-button">Join Now!</button>
```

Comment:
Is this **Join Now!** supposed to be a button or a link to the page?

We should also include more specific text in the call to action for accessibility.

https://accessibility.18f.gov/links/

Reply:
I did have a concern about wrapping buttons with links. Should just be a link to an internal page.